tests/plugins/test_tvplayer.py

```diff
@@ -33,24 +33,48 @@ def test_can_handle_url(self):
     @patch('streamlink.plugins.tvplayer.HLSStream')
     def test_get_streams(self, hlsstream, mock_get_stream_data):
         mock_get_stream_data.return_value = {
-            "stream": "http://test.se/stream1"
+            "response": {"stream": "http://test.se/stream1", "drm": None}
         }
 
         page_resp = Mock()
         page_resp.text = u"""
-                    <div class="video-js theoplayer-skin theo-seekbar-above-controls content-box vjs-fluid"
-                 data-resource= "bbcone"
-                 data-token = "1324567894561268987948596154656418448489159"
-                                    data-content-type="live"
-                    data-environment="live"
-                    data-subscription="free"
-                    data-channel-id="89">
-                <div id="channel-info" class="channel-info">
-                    <div class="row visible-xs visible-sm">
+            <div class="full-bleed-container">
+                <div class="container-fluid">
+                    <div class="row">
+                        <div class="col-xs-12">
+                            <div id="live-player-root"
+                                data-player-library="videojs"
+                                data-player-id="ILWxqLKV91Ql8kF"
+                                data-player-key="2Pw1Eg0Px3Gy9Jm3Ry8Ar5Bi5Vc5Nk"
+                                data-player-uvid="139"
+                                data-player-token="275c808a685a09d6e36d0253ab3765af"
+                                data-player-expiry="1531958189"
+                                data-player-poster=""
+                                data-player-npaw='{"enableAnalytics":true,"username":"","extraparam.1":"TVPlayer","extraparam.2":"","extraparam.3":"free","extraparam.4":"","extraparam.5":"","extraparam.16":null,"content.isLive":true,"content.title":"BBC London News; Weather","content.title2":"","content.duration":"","content.metadata":{"uvid":"e3e45caf-79fe-3827-82b4-ecdfee52acfb","producer":"","status":"","genre":"","thumbnail":"https:\/\/d537y3nbkeq75.cloudfront.net\/tvp\/epg\/e3e45caf-79fe-3827-82b4-ecdfee52acfb.jpg?width=700&lang=en","asset-type":""},"title":"BBC London News; Weather","extraparam.7":"","extraparam.9":"e3e45caf-79fe-3827-82b4-ecdfee52acfb","extraparam.13":null,"extraparam.14":true,"extraparam.6":"","extraparam.8":""}'
```

Comment:
> Shall I just delete that single long line? It would solve the E501 and W605 issues flagged by flake8.
> The content in that line is not required for the test to pass.

Remove both long lines if they're not needed.

However, if there are long JSON payloads in `data-` attributes which are required, then you could also define the JSON object in Python and stringify it in the multiline string, so that it produces the same output. But since they are not needed, it would make the test less legible and that's not the goal here.

Reply:
OK, great.  Many thanks for the info.